dev/custom-pages/home.scss

```diff
@@ -0,0 +1,166 @@
+@import '../custom-base';
+
+.bg-gray {
+  background-color: #efefef;
+}
+
+.home-degree-search-wrap {
+  background-color: #fff
```

Comment:
Athena bg-secondary class could have been used here

Reply:
Same with this style. We're specifically overriding the style underneath in the media query right below this. Defining this here, instead of having to know the `bg-secondary` class is set on the markup, keeps everything all in one place and makes it easier to understand what's going on.